bob/main.py

```diff
@@ -253,17 +254,23 @@ def weather_command(update, context):
 
 def replace_weather_description_with_emojis(description):
     dictionary_of_weather_emojis= {
-        'snow':'\U0001F328',
-        'rain': '\U0001F327',
-        'fog':'\U0001F32B',
-        'smoke':'\U0001F32B',
-        'mist':'\U0001F32B',
-        'clear sky':'\U0001F31E',	
-        'thunderstorm':'\U0001F329',
-        'clouds':'\U00002601'
+        'snow': ['lumisadetta','\U0001F328'],
```

Comment:
Not sure if it could also be like:

english_to_emoji = {'snow': '\U0001F328'}
english_to_finnish = {'snow': 'lumisadetta'}

But don't know if it would be better.